submissions/ses3332002/html-css-popup/index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="build\style.css">
+  <title>Popup window</title>
+</head>
+<body>
+  <header>
+    <div class="user_menu">
```

Comment:
Please, use the exact semantic structure for your menu:
```suggestion
<nav>
  <ul>
    <li><a href="#">Home</a></li>
    <li><a href="#">About</a></li>
    <li><a href="#">Contact</a></li>
  </ul>
</nav>
```

Reply:
It's not that I'm against semantic tags, but there was not any mentioning about the rest of the page elements in the task, so I decide that there could be another element on the page with navigation function (like list of mail folders), so I make my menu just "div", not "nav".

Reply:
As I know, we can use multiple `nav` tags on the page.